docs/ops.html

```diff
@@ -2032,17 +2032,26 @@ <h5 class="anchor-heading"><a id="kafka_streams_store_monitoring" class="anchor-
  </table>
 
   <h5 class="anchor-heading"><a id="kafka_streams_rocksdb_monitoring" class="anchor-link"></a><a href="#kafka_streams_rocksdb_monitoring">RocksDB Metrics</a></h5>
-  All of the following metrics have a recording level of <code>debug</code>.
-  The metrics are collected every minute from the RocksDB state stores.
-  If a state store consists of multiple RocksDB instances as it is the case for aggregations over time and session windows,
-  each metric reports an aggregation over the RocksDB instances of the state store.
+  RocksDB metrics are grouped into statistics-based metrics and properties-based metrics.
+  The former are recorded from statistics that a RocksDB state store collects whereas the latter are recorded from
+  properties that RocksDB exposes.
+  Statistics collected by RocksDB provide cumulative measurements over time, e.g. bytes written to the state store.
+  Properties exposed by RocksDB provide current measurements, e.g., the amount of memory currently used.
   Note that the <code>store-scope</code> for built-in RocksDB state stores are currently the following:
   <ul>
     <li><code>rocksdb-state</code> (for RocksDB backed key-value store)</li>
     <li><code>rocksdb-window-state</code> (for RocksDB backed window store)</li>
     <li><code>rocksdb-session-state</code> (for RocksDB backed session store)</li>
   </ul>
 
+  <strong>RocksDB Statistics-based Metrics:</strong>
+  All of the following statistics-based metrics have a recording level of <code>debug</code> because collecting
+  statistics in <a href="https://github.com/facebook/rocksdb/wiki/Statistics#stats-level-and-performance-costs">RocksDB
+  may have an impact on performance</a>.
+  Statistics-based metrics are collected every minute from the RocksDB state stores.
+  If a state store consists of multiple RocksDB instances, as is the case for aggregations over time and session windows,
```

Comment:
```suggestion
  If a state store consists of multiple RocksDB instances, as is the case for WindowStores and SessionStores,
```
I think we should be more specific here. Also users can have a Window/SessionStore that's unattached to an aggregation, eg in the PAPI, so we should avoid explicitly tying this to aggregations